third_party/imx-secure-enclave/BUILD.gn

```diff
@@ -0,0 +1,184 @@
+# Copyright (c) 2023 Project CHIP Authors
```

Comment:
Throughout: if you add copyright, 2025 is the right year. Do not add NXP as NXP is "CHIP Authors". We do not add every company changing a source code to the history of all files we update, that would be painful (we are a lot of member companies)